Windows 10 Error - Cannot run program "aws"

After installing the aws cli through python, I attempt to run `awscli.bat`, and get the following error:

```
Exception in thread "main" java.io.IOException: Cannot run program "aws": CreateProcess error=2, The system cannot find the file specified
        at java.lang.ProcessBuilder.start(Unknown Source)
        at com.okta.tools.awscli.main(awscli.java:36)
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
        at java.lang.ProcessImpl.create(Native Method)
        at java.lang.ProcessImpl.<init>(Unknown Source)
        at java.lang.ProcessImpl.start(Unknown Source)
        ... 2 more
```

My initial guess was an issue with the way my `PATH` variable was setup, but running `aws` from the command line resolves correctly, and it also looks like the correct directory `C:\Program Files\Python36\Scripts` has been added to the `PATH`.

**Can you all identify anything that I may have missed in order to execute this command correctly?**

Below is execution of `aws` from Powershell with Admin access. The same result also occurs from my user-level account.
```
PS C:\> aws
usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]
To see help text, you can run:

  aws help
  aws <command> help
  aws <command> <subcommand> help
aws: error: the following arguments are required: command
```

This is a snapshot of my system-level environment variables. I have included the Python paths in both User and System.

![image](https://user-images.githubusercontent.com/5298126/36917105-846b8260-1e1b-11e8-8fc2-2e69d4295cce.png)

:-1: me too.
```
c:\Users\dlgoodr\.okta>java -classpath .\okta-aws-cli-1.0.1.jar com.okta.tools.awscli
Username: dlgoodr
Password:

Please choose the role you would like to assume:
[excised]
Account: [x]
[excised]
        [ 4 ]: TEST
[excised]
Selection: 4
Exception in thread "main" java.io.IOException: Cannot run program "aws": CreateProcess error=2, The system cannot find the file specified
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
        at com.okta.tools.awscli.main(awscli.java:39)
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
        at java.base/java.lang.ProcessImpl.create(Native Method)
        at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:420)
        at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:151)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)
        ... 2 more

c:\Users\dlgoodr\.okta>aws
usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]
To see help text, you can run:

  aws help
  aws <command> help
  aws <command> <subcommand> help
aws: error: too few arguments

c:\Users\dlgoodr\.okta>which aws
/c/Python27/Scripts/aws

c:\Users\dlgoodr\.okta>
```

**Switching Role with withokta**

It appears to be rewriting my profile, changing the account number in the `role_arn` and the `source_profile` value.
```
PS C:\Users\dlgoodr\.okta> $env:OKTA_PROFILE = "target-account-profile"
PS C:\Users\dlgoodr\.okta> .\withokta.bat cmd /c "aws --profile target-account-profile iam list-roles"

C:\Users\dlgoodr\.okta>java -classpath "C:\Users\dlgoodr\.okta\*" com.okta.tools.WithOkta cmd /c "aws --profile target-account-profile iam list-roles"

An error occurred (AccessDenied) when calling the AssumeRole operation: User: arn:aws:sts::<landing_account>:assumed-role/myrole/dlgoodr@ad.corp.com is not authorized to perform: sts:AssumeRole on resource: arn:aws:iam::<landing_account>:role/myrole
PS C:\Users\dlgoodr\.okta>
```

my `.aws\config` is:
```
[profile default]
output = json
region = us-east-1

[profile target-account-profile]
region = us-east-1
role_arn = arn:aws:iam::<landing_account>:role/myrole
source_profile = target-account-profile_source
```
before I ran withokta, <landing_account> was <target_account> and `source_profile` was something entirely different so i'm very confused about how this should work.  Never prompted for creds, either, so ... it's either very broken or I have no idea how to use it.  I'm ok with it being the second, but guidance would be helpful.


**Using the .bat file** instead of calling java explicitly doesn't change the behavior.
```


C:\Users\dlgoodr\.okta>awscli.bat

C:\Users\dlgoodr\.okta>rem

C:\Users\dlgoodr\.okta>rem Copyright 2017 Okta

C:\Users\dlgoodr\.okta>rem

C:\Users\dlgoodr\.okta>rem Licensed under the Apache License, Version 2.0 (the "License");

C:\Users\dlgoodr\.okta>rem you may not use this file except in compliance with the License.

C:\Users\dlgoodr\.okta>rem You may obtain a copy of the License at

C:\Users\dlgoodr\.okta>rem

C:\Users\dlgoodr\.okta>rem     http://www.apache.org/licenses/LICENSE-2.0

C:\Users\dlgoodr\.okta>rem

C:\Users\dlgoodr\.okta>rem Unless required by applicable law or agreed to in writing, software

C:\Users\dlgoodr\.okta>rem distributed under the License is distributed on an "AS IS" BASIS,

C:\Users\dlgoodr\.okta>rem WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.

C:\Users\dlgoodr\.okta>rem See the License for the specific language governing permissions and

C:\Users\dlgoodr\.okta>rem limitations under the License.

C:\Users\dlgoodr\.okta>rem

C:\Users\dlgoodr\.okta>java -classpath "C:\Users\dlgoodr\.okta\*" com.okta.tools.awscli
Username: dlgoodr
Password:
Exception in thread "main" java.io.IOException: Cannot run program "aws": CreateProcess error=2, The system cannot find the file specified
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
        at com.okta.tools.awscli.main(awscli.java:39)
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
        at java.base/java.lang.ProcessImpl.create(Native Method)
        at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:420)
        at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:151)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)
        ... 2 more

C:\Users\dlgoodr\.okta>




```
**1.0.2** doesn't appear to be available yet, but I'll happily try it when it's available.

--- a/bin/Install-OktaAwsCli.ps1
+++ b/bin/Install-OktaAwsCli.ps1
@@ -82,7 +82,7 @@ function Okta-ListRoles {
 }
 function okta-aws {
     Param([string]$Profile)
-    With-Okta -Profile $Profile aws --profile $Profile @args
+    With-Okta -Profile $Profile ((Get-Command aws).Name) --profile $Profile @args
 }
 function okta-sls {
     Param([string]$Profile)

